wordpress-mobile/WordPress-Android
# Feature/use view binding in pages

This PR introduces view bindings in pages activity/fragment, page parent activity/fragment and page parent search.
The changes are quite straightforward

To test:
- Go to Site pages
- Check it looks as expected
- Click on an overflow menu on a page -> "Set parent"
- Check the "Page parent" screen looks as expected
- Click on "Search"
- Check everything works

PR submission checklist:
